### ISSUE TITLE
How to run esformatter? Does it have role in build?

### ISSUE THREAD

**Issue by mccalluc** [refinery-platform/refinery-platform#1952]
ESFormatter was introduced in #982.

- How is it run? It seems to be missing a dependency:
```
$ grunt esformatter **/*
Warning: Task "bower_components/angular" not found. Use --force to continue.
Aborted due to warnings.
```

- Does it, or should it have any role in the build? I don't think it does right now, but maybe it should, or maybe I'm missing something?

**Comment by flekschas** [comment]
@mccalluc Used some time ago to auto-format code. Feel free to remove

**Comment by mccalluc** [comment]
(Scott is on board with removing this.)

**Comment by mccalluc** [comment]
Fixed by #1962
